text/0000-fix-fees.md

```diff
@@ -31,6 +33,7 @@ We therefore want the minimum relay fee be proportional to the weight of the tra
 We can calibrate the fee system by stipulating that creating one output should cost at least one Grin-cent (formerly 0.4 Grin-cent).
 Linearity is another desirable property, which for instance allows the two parties in a payjoin to each pay their own input and output fees,
 while splitting the kernel fee.
+The least significant bits will specify a minimum fee overpayment factor, preventing aggregation with lesser overpaying transactions.
```

Comment:
Just to clarify - This is effectively splitting the fee (8 bytes) into two separate fields?
If we use, for example, the least significant full byte for fee factor then the fee is the remaining 7 bytes? 
But with these 7 bytes representing the most significant 7 bytes of a full 8 byte value, with the least significant byte handled as all zeros?

And if this least significant byte is all zeros then no fee factor is involved?

Reply:
No, the fee is still the full 8 bytes. The special interpretation of the least significant byte is outside the consensus model. Only used for relay/mempool acceptance.
In your example, the fee is 32770 nanogrin (2^15+2^1) and it should be at least 3x larger than the minimal required fee for whatever tx this kernel is in.

Reply:
Why `factor + 1` and not just `factor`? In this example why 3x?

Reply:
It makes no sense to have the fees be 0 times the minimum required. So fee factors are necessarily >= 1. Because the least significant bits are normally 0, it's only natural to map 0 bits to a fee factor of 1, preserving the current behaviour.